27.md

```diff
@@ -0,0 +1,18 @@
+NIP-27
+======
+
+Text Note References
+--------------------
+
+`draft` `optional` `author:fiatjaf` `author:scsibug` `author:arthurfranca`
+
+This document standardizes the treatment given by clients of inline references of other events and profiles inside the content of `text notes` (currently kinds 1 and 30023).
+
+Once a mention is identified by a client, for example, when an user pastes a NIP-19 nostr profile string inside the event content input field (or optionally by any other means the client may wish to support, such as selecting an user from a context menu or autocompleter), the client MUST replace it with the notation `nostr:nprofile1qqsw3dy8cpu...6x2argwghx6egsqstvg` as per NIP-21 before effectively creating the event. The client MAY add the corresponding pubkey to the `.tags` with the tag `p` if it wishes to notify mentioned user.
+
+The same process applies for referencing event IDs (using `nostr:nevent1...`), except that the client SHOULD NOT add the event id to the `.tags` with the tag `e` so to not negatively impact thread loading.
```

Comment:
should be "(using `nostr:note1...`)" instead of "(using `nostr:nevent1...`)"

Reply:
Oh, I apologize. Obviously my knowledge of NIP-19 was not up to speed!

Reply:
Shouldn't the NIP be agnostic as to what is given there? There is also `naddr1...`.

Reply:
yeah I guess. but kinda seems good to nudge everyone into including relay data as would make everything more reliable/targeted.

naddr1 is a good call